powershell-adapter/psDscAdapter/powershell.resource.ps1

```diff
@@ -66,6 +66,11 @@ if ($jsonInput) {
     if ($jsonInput -ne '@{}') {
         $inputobj_pscustomobj = $jsonInput | ConvertFrom-Json
     }
+    else 
+    {
+        $jsonInput = $jsonInput | ConvertTo-Json
```

Comment:
Can you please give examples of `expected behavior` and `actual behavior`?

Reply:
Whenever a user tries to do a get operation on a resource without specifying any input, it should give a proper error message to the user. That's it actually. If it can be captured higher level, that would also do the trick.